aioftp/client.py

```diff
@@ -547,13 +697,38 @@ def list(self, path="", *, recursive=False):
 
             >>> stats = await client.list()
         """
-        class AsyncClientLister(AsyncListerMixin):
+        class AsyncListerClient(AsyncListerMixin):
```

Comment:
Ooops, was doing some heavy reformatting and deleted the class, forgot what the name was :(